functions/_pure_prompt_ssh_user.fish

```diff
@@ -1,5 +1,5 @@
 function _pure_prompt_ssh_user
-    set --local username (id --user --name) # current user name
+    set --local username (id -u -n) # current user name
```

Comment:
👍

Reply:
I have fixed a few issue in the macOS CI, but still too many of them. 
I will release the fix without full macOS CI validation